index.rst

```diff
@@ -250,6 +251,7 @@ Environmental
 .. imgtable::
 
     AHT10, components/sensor/aht10, aht10.jpg, Temperature & Humidity
+    AirThings BLE, components/sensor/airthings_ble, airthings_logo.png, Radon, Temperature & Humidity & Pressure
```

Comment:
```suggestion
    AirThings BLE, components/sensor/airthings_ble, airthings_logo.png, Temperature & Humidity & Pressure
```

Reply:
Radon reading really is supported, even if the class is not there.

Reply:
Ah, Airthings is in the home page in two places, `Air Quality` and `Environment`. I was not too sure which one Radon should come under, and I put it under the `Air Quality` header as it was quite long under `Environment`... 

Feel free to make another PR to update the specific classifications. =)